### DISCUSSION TITLE
concore-editor 1.0.0 released!

### DISCUSSION THREAD

**Discussion by pradeeban** [ControlCore-Project/concore-editor#143] [Announcements]
Version 1.0.0 with contributions to the main branch made until March 9th, 2023.
* concore editor forked [DHGWorkflow](https://github.com/NISYSLAB/DHGWorkflow) as an editor for concore in 2022.
* bug fixes to concore editor from the volunteer contributors in 2023.

<hr /><em>This discussion was created from the release <a href='https://github.com/ControlCore-Project/concore-editor/releases/tag/1.0.0'>1.0.0</a>.</em>

**Comment by parteekcoder** [comment]
Congratulations @pradeeban for this release

**Reply by pradeeban** [reply]
Thank you for your contributions and bug fixes.

**Comment by Rahuljagwani** [comment]
Congratulations @pradeeban and everyone involved in development of Concore-Editor. In future, we will even work harder and smarter to make it success.

**Reply by pradeeban** [reply]
Thank you for your bug fixes and contributions to CONTROL-CORE projects.
